meetings/2022-03-14.md

```diff
@@ -0,0 +1,108 @@
+# W3C Solid Community Group: Authentication Panel
+
+* Date: 2022-03-14T15:00:00Z
+* Call: https://meet.jit.si/solid-authentication
+* Chat: https://gitter.im/solid/authentication-panel
+* Repository: https://github.com/solid/authentication-panel
+
+
+## Present
+* Aaron Coburn
+* elf Pavlik
+* Matthieu Bosquet
+* Nicolas Seydoux
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* elf Pavlik
+---
+
+## Agenda
+
+### Pull Requests
+https://github.com/solid/solid-oidc/pulls
+
+* 
+
+### Issues
+https://github.com/solid/solid-oidc/issues
+
+* [In some cases OIDC issuer can't be disclosed in WebID Profile #92](https://github.com/solid/solid-oidc/issues/92)
+
+
+## Minutes
+
+### DST changess
+
+* Pavlik: Can we stick with US time for next two weeks?
+* Matthieu: ok for me, even better.
+* Nicolas: ok
+
+
+### In some cases OIDC issuer can't be disclosed in WebID Profile 
+
+[Issue#92](https://github.com/solid/solid-oidc/issues/92)
+
+* MB: During AuthZ this requirement came up. We also discussed TrustedApps, which has privacy issue of having user disclosing trusted apps to the public. It stroke me that in many places disclosing trusted issuers in WebID document could be privacy issue. Depending on circumstances it could be damaging to someone forced to disclose it. For example government service.
```

Comment:
```suggestion
* MB: During the [AuthZ panel](https://github.com/solid/authorization-panel/blob/main/meetings/2022-03-02.md#default-client-based-restrictions) this requirement came up. We also discussed the similar [WebID profile TrustedApps](https://github.com/solid/web-access-control-spec/blob/main/README-v0.5.0.md#adding-trusted-web-apps), which has the privacy issue of having a user disclosing their trusted apps to the public. It stroke me that in many places disclosing trusted issuers in WebID document could be privacy issue too. Depending on circumstances it could be damaging to someone forced to disclose them. For example a government service.
```